src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,21 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+export const PostInfo = ({ post, user, comments }) => {
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{post.title}</h3>
+        <p>
+          {' Posted by '}
+          <UserInfo user={user} />
```

Comment:
<img width="527" alt="image" src="https://github.com/user-attachments/assets/4869daa6-7f95-43fb-a760-84170da2ebe5">
find can return an undefined, so it's better to add the check

```suggestion
          {user && <UserInfo user={user} />}
```